.github/actions/publish_doc/action.yml

```diff
@@ -18,6 +18,10 @@ inputs:
     description: 'Spark version 3.2.X'
     required: true
     default: "3.2"
```

Comment:
```suggestion
     default: "3.3"
```

Reply:
Why change the 3.2 version to the value 3.3?